cloudfront/iiif.wellcomecollection.org/tests/requirements.txt

```diff
@@ -0,0 +1,2 @@
+click
+requests
```

Comment:
I would recommend renaming this to `requirements.in`, and using `pip-tools` to create a `requirements.txt` that freezes a known-good set of dependencies. That way this has more chance of continuing to work in future, rather than breaking when one of these packages releases a new version.

See https://hynek.me/articles/python-app-deps-2018/#pip-tools--everything-old-is-new-again

Reply:
I'm familiar, but didn't go to the extra trouble here. It probably is worth it 👍

Reply:
I had difficulty getting pip-tools to work for me, so i've fixed the versions in requirements.txt.